src/app/child-dev-project/children/children.service.ts

```diff
@@ -197,13 +233,35 @@ export class ChildrenService {
         },
       },
     };
+
+    // TODO: remove these and use general note_by_relatedEntities instead --> to be decided later #1501
     // creating a by_... view for each of the following properties
     ["children", "schools", "authors"].forEach(
       (prop) =>
         (designDoc.views[`by_${prop}`] = this.createNotesByFunction(prop))
     );
 
-    return this.dbIndexing.createIndex(designDoc);
+    await this.dbIndexing.createIndex(designDoc);
+
+    const newDesignDoc = {
+      _id: "_design/notes_related_index",
+      views: {
+        note_by_relatedEntities: {
+          map: `(doc) => {
+            if (!doc._id.startsWith("${Note.ENTITY_TYPE}") && !doc._id.startsWith("${EventNote.ENTITY_TYPE}")) return;
```

Comment:
This is the only view where Notes and EventNotes are emitted. What is the reason for that?

I am a bit worried that this inconsistency will cause unexpected behaviour when using the views.

Reply:
I think this is left over from the initial draft, where we considered merging a lot of indices. Removing EventNotes here for now ...

Reply:
there is some risk that a user edits and event (EventNote) through our note details view, adds a relatedEntity and then this will not show up. But this has been a gap for the other references of EventNote already anyway